Entry 2 -- November 8.md

```diff
@@ -15,12 +17,21 @@ design process.
 you need to make, what implementation issue are you trying to solve, or how
 are you evaluating your design and implementation?**
 
+I feel like my API is rudimentary at best. I spent a lot of time this week writing and rewriting the API structure because I couldn't find something I was *too* happy with. Any suggestions in this department would be helpful. Also, ScalaIDE/Eclipse dislikes my program structure and is unable to recognize class names in the same package. I spent some time trying to fix this issue but I think I will have to go to office hours/grutoring to try and fix this one.
```

Comment:
It could help your critique partners (and yourself!) to provide more detail about these two issues. 

For the design:
   + What specifically were you unhappy about? 
   + Have you developed design principles that aren't being satisfied by the implementation? 
   + Or, do you feel like the principles are changing _as_ you implement?

For the ScalaIDE issues, what errors were you getting? A screen shot can help. Also, posting on Piazza can help you make progress earlier. Don't feel like to you have to figure out all those things on your own!